apps/nowcasting-app/components/charts/gsp-pv-remix-chart/index.tsx

```diff
@@ -33,6 +38,19 @@ const GspPvRemixChart: FC<{
     gspForecastData?.find((fc) => formatISODateString(fc?.targetTime) === selectedTime) ||
     ({} as any);
   const pvPercentage = (forcastAtSelectedTime.expectedPowerGenerationNormalized || 0) * 100;
+
+  // set ymax to the installed capacity of the graph
+  var yMax = gspInfo?.installedCapacityMw;
```

Comment:
use `let` instead of `var` , because var does not respect code block scope only function scope
(i should've put this in eslint :))

Reply:
Thanks, but does the code a few lines below mean we need to use var, as the variable is redefined?

Reply:
Thanks, yea 'let' works. I must have tried 'const' before (and that didnt work)